pillar/heroku/xpro.sls

```diff
@@ -102,6 +104,7 @@
       'MITXPRO_BASE_URL': 'https://xpro.mit.edu',
       'MITXPRO_SECURE_SSL_HOST': 'xpro.mit.edu',
       'ENABLE_ORDER_RECEIPTS': True,
+      'FEATURE_ENABLE_ENTERPRISE': False,
```

Comment:
I think we can enable this as well. This would not be auto-deployed but once we test it on RC we can take these configs to prod without a new PR.

Reply:
Enabled in https://github.com/mitodl/salt-ops/pull/1613/commits/b7631f46f4b231874c648b04cb238694a944a251